README.md

```diff
@@ -97,54 +141,15 @@ Build and install [RELIC](https://github.com/relic-toolkit/relic)
     make
     sudo make install
 
-Build and install [cryptopp](https://github.com/weidai11/cryptopp)
-
-    cd
-    git clone https://github.com/weidai11/cryptopp.git
-    cd cryptopp/
-    git checkout CRYPTOPP_8_2_0;
-    make
-    sudo make install
-
-Get GNU Parallel
-
-    sudo apt-get install parallel
-
 Get g++:
```

Comment:
Shouldn't g++ be installed before conan, so it can compile missing dependencies?